### PR TITLE
submit forms to update cases

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?173809

@czue 

buddy @TylerSheffels

### CODE REVIEW

File: custom/bihar/management/commands/bihar_cleanup_case_2.py [hunk 1]
```diff
@@ -1,69 +1,120 @@
...
+        if case:
+            return case
+        else:
+            try:
+                case = CommCareCase.get(case_id)
+            except ResourceNotFound:
+                print "** Error: could not find case with id " + case_id
+
+        return CaseToUpdate(case)
+
```
Comment (dimagimon) [inline custom/bihar/management/commands/bihar_cleanup_case_2.py]: W293 blank line contains whitespace

File: custom/bihar/management/commands/bihar_cleanup_case_2.py [hunk 2]
```diff
@@ -1,69 +1,120 @@
...
+                reader = csv.reader(f)
+                reader.next()
+                for row in reader:
+                    case_id = row[0]
+                    to_update = self.get_case_to_update(case_id)
+                    owner_id = row[1]
+                    if to_update and to_update.case.owner_id != owner_id:
+                        to_update.new_owner_id = owner_id
+                        self.cases_to_update[case_id] = to_update
+                        print("Updated case with id " + to_update.case.case_id + " to have owner with id " + to_update.case.owner_id)
```
Comment (dimagimon) [inline custom/bihar/management/commands/bihar_cleanup_case_2.py]: E501 line too long (133 > 115 characters)

File: custom/bihar/management/commands/bihar_cleanup_case_2.py [hunk 3]
```diff
@@ -1,69 +1,120 @@
...
+                reader = csv.reader(f)
+                reader.next()
+                for row in reader:
+                    case_id = row[0]
+                    to_update = self.get_case_to_update(case_id)
+                    if to_update:
+                        if to_update.case.user_id == MOTECH_ID:
+                            to_update.new_type = "task"
+                            self.cases_to_update[case_id] = to_update
+                            print("Case with name " + to_update.case.name.encode('ascii', 'xmlcharrefreplace') + " updated to type " + to_update.case.type)
```
Comment (dimagimon) [inline custom/bihar/management/commands/bihar_cleanup_case_2.py]: E501 line too long (155 > 115 characters)

File: custom/bihar/management/commands/bihar_cleanup_case_2.py [hunk 4]
```diff
@@ -1,69 +1,120 @@
...
+                        else:
+                            print("Type not updated for case with name " + to_update.case.name.encode('ascii', 'xmlcharrefreplace'))
```
Comment (dimagimon) [inline custom/bihar/management/commands/bihar_cleanup_case_2.py]: E501 line too long (132 > 115 characters)

**Comment by snopoke** [comment]
@nickpell @orangejenny FYI

**Comment by TylerSheffels** [comment]
Looks good to me, but want another +1 before I merge.

File: custom/bihar/management/commands/bihar_cleanup_case_2.py (hunk 3 above)
Comment (nickpell) [inline custom/bihar/management/commands/bihar_cleanup_case_2.py]: might prefer case id

File: custom/bihar/management/commands/bihar_cleanup_case_2.py (hunk 4 above)
Comment (nickpell) [inline custom/bihar/management/commands/bihar_cleanup_case_2.py]: also might prefer to print case id

**Comment by nickpell** [comment]
made some non-blocking comments

**Comment by czue** [comment]
build fail not related

File: custom/bihar/management/commands/bihar_cleanup_case_2.py (hunk 3 above)
Comment (czue) [inline custom/bihar/management/commands/bihar_cleanup_case_2.py]: agree with this, but not blocking. will merge but could be worth changing

File: custom/bihar/management/commands/bihar_cleanup_case_2.py (hunk 4 above)
Comment (snopoke) [inline custom/bihar/management/commands/bihar_cleanup_case_2.py]: https://github.com/dimagi/commcare-hq/pull/7293/files

**Pull request merged**